package.json

```diff
@@ -6,14 +6,15 @@
     "generate-hook": "nx generate open-feature hook",
     "generate-provider": "nx generate open-feature provider",
     "test": "nx run-many --all --target=test --skip-nx-cache",
-    "e2e": "nx run-many --all --target=e2e --skip-nx-cache --output-style=stream --parallel=false",
+    "e2e": "nx run-many --all --exclude js-sdk-contrib --target=e2e --skip-nx-cache --output-style=stream --parallel=false",
```

Comment:
This fixed the weird loop (removing the root project from the `--all`) no idea why this is suddenly necessary.